.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
+language: ruby
+rvm:
+  - 2.3.1
+services:
+  - docker
+
+script:
+  - CD/deploy.sh
```

Comment:
What does `CD` stand for?  Continuous Delivery/Deployment?  It's a bit more terse than it maybe needs to be...